app/models/job.rb

```diff
@@ -10,13 +10,24 @@ class Job < ActiveRecord::Base
 
   after_update :notify_if_published
 
-  scope :published, lambda {
-    where("jobs.published_at IS NOT NULL AND jobs.published_at <= ?", Time.zone.now)
-  }
+  class << self
+    # Jobs that should appear in public listings
+    #
+    # @returns [ActiveRecord::Relation<Job>] active jobs
 
-  scope :stale, lambda {
-    where("created_at < ?", 60.days.ago)
-  }
+    def active
+      published.where("jobs.published_at >= ?", 60.days.ago)
+    end
```

Comment:
This is the heart of the change. Now, jobs are show publicly based on if they are active, not just on if they are published.

- If the published_at date is in the future, it would not show up in the jobs listing because it's not active yet.
- If the published_at date is more than 60 days ago, it's no longer shown because it's no longer active
- If the published date is after today but less or equal to 60 days ago, then it's shown